packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -407,10 +407,30 @@ export const Checkout = ({
             plan={selectedPlan}
             onClick={() => setCheckboxSet(!checkboxSet)}
             type={PaymentMethodHeaderType.SecondStep}
-            invoice={invoice}
           />
 
-          <>
+          <div
+            className={`mt-8 ${
+              !checkboxSet ? 'payment-form-disabled cursor-not-allowed' : ''
+            }`}
+            tabIndex={!checkboxSet ? 0 : undefined}
+            aria-disabled={!checkboxSet}
+            aria-live="polite"
+            onClick={
+              !checkboxSet
+                ? (event) => {
+                    event.preventDefault();
+                  }
+                : undefined
+            }
+            onKeyPress={
+              !checkboxSet
+                ? (event) => {
+                    if (event.key !== 'Tab') event.preventDefault();
+                  }
+                : undefined
+            }
+          >
```

Comment:
Unlike `AcceptedCards` this change poses a problem. `PayPalButton` has a loading state. While I agree that nothing should appear if for whatever reason the button doesn't appear, it might look a little strange to the customer to have everything appear. At least with the title, the customer would be aware that option is supposed to exist and be available for them.

Since this is a design/UX change, I think we should keep it as it is now and update in 3.0. Thoughts?

Reply:
Aligned, can we update the PayPalButton component to remove the nested div/combine the classNames?

Reply:
Haha yes! Fully aligned. Thanks for the catch!